docs/pages/store/encoding.mdx

```diff
@@ -239,7 +239,7 @@ enum SchemaType {
 Example: `Schema` with 3 fields (`uint64`, `uint40`, `address[]`).
 
 ```
-0x 000D 02 01 0704c300000000000000000000000000000000000000000000000000
+0x 000d 02 01 0704c300000000000000000000000000000000000000000000000000
```

Comment:
did you mean to change this line? https://github.com/latticexyz/mud/blob/0b8e19ff52e2ea4c8bf721e20549dd415df4b9a1/docs/pages/store/encoding.mdx#L328

Reply:
Oops, yes. I fixed it, and also changed `0808` to `0805`.